deebot_client/commands/json/stats.py

```diff
@@ -51,3 +52,14 @@ def _handle_body_data_dict(
         stats_event = TotalStatsEvent(data["area"], data["time"], data["count"])
         event_bus.notify(stats_event)
         return HandlingResult.success()
+
+
+class GetOnStats(OnStats, JsonCommandWithMessageHandling):
+    """Get onStats command."""
+
+    NAME = "getOnStats"
```

Comment:
Are you sure this is a valid command? Did you test your code?

Reply:
I didn’t test this code, I’ve no idea on how to test it on my T9.
Is there a way to gather the command list directly from the device ? 

anyway, I added it to have a command into device’s capability, otherwise I’ve no idea on how to “link” mqtt data to the device’s class, can you point me on how to do that?

Reply:
> Is there a way to gather the command list directly from the device ?

Unfortunately not. We can only use trial and error.
You can checkout the example of the readme and test if you get an answer from your bot to test the command

Reply:
I've removed this command, because those stats come from mqtt topic "iot/atr/onStats/" and I've seen device subscribes to topics; let me know if commands are needed to handle mqtt messages.